packages/aura-language-server/src/tern-server/tern-server.ts

```diff
@@ -70,107 +69,30 @@ const defaultConfig = {
     dependencyBudget: 20000,
 };
 
-function readJSON(fileName): any {
-    const file = fs.readFileSync(fileName, 'utf-8');
-    try {
-        return JSON.parse(file);
-    } catch (e) {
-        console.warn('Bad JSON in ' + fileName + ': ' + e.message);
-    }
-}
-
-function findDefs(libs): any[] {
-    const ternlibpath = require.resolve('../tern/lib/tern');
-    const ternbasedir = path.join(ternlibpath, '..', '..');
-
-    const defs = [];
-    const src = libs.slice();
-    for (let file of src) {
-        console.log(`Loading support library: ${file}`);
-        if (!/\.json$/.test(file)) {
-            file = file + '.json';
-        }
-        const def = path.join(ternbasedir, 'defs', file);
-        if (fs.existsSync(def)) {
-            defs.push(readJSON(def));
-        } else {
-            console.log(`Not found: ${file}`);
-        }
-    }
-    return defs;
-}
-
-async function loadLocal(plugin, rootPath): Promise<boolean> {
-    let found;
-    try {
-        // local resolution only here
-        found = require.resolve('./tern-' + plugin);
-    } catch (e) {
-        return false;
-    }
-
-    const mod = await import(found);
-    if (mod.hasOwnProperty('initialize')) {
-        mod.initialize(rootPath);
-    }
-    return true;
-}
-
-async function loadBuiltIn(plugin: string, rootPath: string): Promise<boolean> {
-    const ternlibpath = require.resolve('../tern/lib/tern');
-    const ternbasedir = path.join(ternlibpath, '..', '..');
-
-    const def = path.join(ternbasedir, 'plugin', plugin);
+async function loadPlugins(): Promise<{ aura: true; modules: true; doc_comment: true }> {
+    await import('./tern-aura');
```

Comment:
there were 3 files loaded dynamically, always the same ones.  They're static now.  Adding more (YAGNY, it's Aura) isn't much harder this than adding them to the plugins list.